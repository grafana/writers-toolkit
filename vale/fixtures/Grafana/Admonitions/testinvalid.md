Don't use:

- > **note:**
- > **note**:
- > **Note:**
- > **Note**:
- > **NOTE:**
- > **NOTE**:

- > **caution:**
- > **caution**:
- > **Caution:**
- > **Caution**:
- > **CAUTION:**
- > **CAUTION**:

- > **warning:**
- > **warning**:
- > **Warning:**
- > **Warning**:
- > **WARNING:**
- > **WARNING**:

- > **tip:**
- > **tip**:
- > **Tip:**
- > **Tip**:
- > **TIP:**
- > **TIP**:
