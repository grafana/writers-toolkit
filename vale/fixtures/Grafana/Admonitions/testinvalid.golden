testinvalid.md:3:3:Grafana.Admonitions:Prefer the `admonition` shortcode over blockquotes. The admonition shortcode renders its content in a blockquote with consistent styling across the website. 
testinvalid.md:4:3:Grafana.Admonitions:Prefer the `admonition` shortcode over blockquotes. The admonition shortcode renders its content in a blockquote with consistent styling across the website. 
testinvalid.md:5:3:Grafana.Admonitions:Prefer the `admonition` shortcode over blockquotes. The admonition shortcode renders its content in a blockquote with consistent styling across the website. 
testinvalid.md:6:3:Grafana.Admonitions:Prefer the `admonition` shortcode over blockquotes. The admonition shortcode renders its content in a blockquote with consistent styling across the website. 
testinvalid.md:7:3:Grafana.Admonitions:Prefer the `admonition` shortcode over blockquotes. The admonition shortcode renders its content in a blockquote with consistent styling across the website. 
testinvalid.md:8:3:Grafana.Admonitions:Prefer the `admonition` shortcode over blockquotes. The admonition shortcode renders its content in a blockquote with consistent styling across the website. 
testinvalid.md:10:3:Grafana.Admonitions:Prefer the `admonition` shortcode over blockquotes. The admonition shortcode renders its content in a blockquote with consistent styling across the website. 
testinvalid.md:11:3:Grafana.Admonitions:Prefer the `admonition` shortcode over blockquotes. The admonition shortcode renders its content in a blockquote with consistent styling across the website. 
testinvalid.md:12:3:Grafana.Admonitions:Prefer the `admonition` shortcode over blockquotes. The admonition shortcode renders its content in a blockquote with consistent styling across the website. 
testinvalid.md:13:3:Grafana.Admonitions:Prefer the `admonition` shortcode over blockquotes. The admonition shortcode renders its content in a blockquote with consistent styling across the website. 
testinvalid.md:14:3:Grafana.Admonitions:Prefer the `admonition` shortcode over blockquotes. The admonition shortcode renders its content in a blockquote with consistent styling across the website. 
testinvalid.md:15:3:Grafana.Admonitions:Prefer the `admonition` shortcode over blockquotes. The admonition shortcode renders its content in a blockquote with consistent styling across the website. 
testinvalid.md:17:3:Grafana.Admonitions:Prefer the `admonition` shortcode over blockquotes. The admonition shortcode renders its content in a blockquote with consistent styling across the website. 
testinvalid.md:18:3:Grafana.Admonitions:Prefer the `admonition` shortcode over blockquotes. The admonition shortcode renders its content in a blockquote with consistent styling across the website. 
testinvalid.md:19:3:Grafana.Admonitions:Prefer the `admonition` shortcode over blockquotes. The admonition shortcode renders its content in a blockquote with consistent styling across the website. 
testinvalid.md:20:3:Grafana.Admonitions:Prefer the `admonition` shortcode over blockquotes. The admonition shortcode renders its content in a blockquote with consistent styling across the website. 
testinvalid.md:21:3:Grafana.Admonitions:Prefer the `admonition` shortcode over blockquotes. The admonition shortcode renders its content in a blockquote with consistent styling across the website. 
testinvalid.md:22:3:Grafana.Admonitions:Prefer the `admonition` shortcode over blockquotes. The admonition shortcode renders its content in a blockquote with consistent styling across the website. 
testinvalid.md:24:3:Grafana.Admonitions:Prefer the `admonition` shortcode over blockquotes. The admonition shortcode renders its content in a blockquote with consistent styling across the website. 
testinvalid.md:25:3:Grafana.Admonitions:Prefer the `admonition` shortcode over blockquotes. The admonition shortcode renders its content in a blockquote with consistent styling across the website. 
testinvalid.md:26:3:Grafana.Admonitions:Prefer the `admonition` shortcode over blockquotes. The admonition shortcode renders its content in a blockquote with consistent styling across the website. 
testinvalid.md:27:3:Grafana.Admonitions:Prefer the `admonition` shortcode over blockquotes. The admonition shortcode renders its content in a blockquote with consistent styling across the website. 
testinvalid.md:28:3:Grafana.Admonitions:Prefer the `admonition` shortcode over blockquotes. The admonition shortcode renders its content in a blockquote with consistent styling across the website. 
testinvalid.md:29:3:Grafana.Admonitions:Prefer the `admonition` shortcode over blockquotes. The admonition shortcode renders its content in a blockquote with consistent styling across the website. 
